docs/section11/Rule11-8.md

```diff
@@ -0,0 +1,65 @@
+    - look at each service water heating use in the BAT: `for swh_use_id in swh_bats_and_uses_p[swh_bat]:`
+      - get the swh use from the swh_use_id: `swh_use = get_component_by_id(P_RMD, swh_use_id)`
+      - check if there are SWH loads in this use, continue to rule logic.  In this case, we are not worried about the use_units, we just want to know that there is a use that is greater than 0: `if swh_use.use > 0: CONTINUE TO RULE LOGIC`
+    - otherwise, rule is not applicable for this swh_bat: `else: NOT_APPLICABLE`
+
+        ## Rule Logic: 
+    - use get_SWH_equipment_associated_with_each_SWH_bat to get the SWH use types and SWH use in the building: `swh_bats_and_equip_dict = get_SWH_equipment_associated_with_each_SWH_bat(B_RMD)`
+    - get a dictionary of all SWH equipment connected to the current SWH bat in the baseline model: `swh_bats_and_equip_dict_this_use = swh_bats_and_equip_dict[swh_bat]`
+    - get the quantity of service water heating distribution systems serving this building area type: `num_swh_systems = len(swh_bats_and_equip_dict_this_use["SWHDistribution"])`
+    - get the quantity of service water heating equipment serving this building area type: `num_swh_equipment_this_use = len(swh_bats_and_equip_dict_this_use["SWHHeatingEq"])`
+    - create a boolean is_referenced_in_other_bats and set it to false: `is_referenced_in_other_bats = false`
+    
+    - if there is exactly one system, check that the system is not used in any of the other BATs: `if num_swh_systems == 1:`
+        - create a variable swh_dist_id and set it to the id of the swh distribution system for this bat: `swh_dist_id = swh_bats_and_equip_dict_this_use["SWHDistribution"][0]`
+        - loop through each bat: `for other_swh_bat in swh_bats_and_equip_dict:`
+        - if other_swh_bat is not equal to the current swh_bat: `if other_swh_bat != swh_bat:`
+            - check whether the swh_dist_id is in the list of SWH Distribution system for the other SWH_bat: `if swh_dist_id in? swh_bats_and_equip_dict[other_swh_bat]["SWHDistribution"]:`
+            - set the boolean is_referenced_in_other_bats to true: `is_referenced_in_other_bats = true`
+    - create a boolean called multiple_segments_with_bat_other and set it to false.  This boolean will tell us if the a) the swh_bat is "OTHER"; b) there are multiple building segments with swh_bat equal to "OTHER": `multiple_segments_with_bat_other = false` 
+    - check whether the swh_bat is "OTHER": `if swh_bat == "OTHER":`
+        - now create a variable num_segments and set it to 0.  This variable will be used to count the number of building segments with swh_bat == "OTHER": `num_segments = 0`
+        - now look at each building segment in the building: `for building_segment in B_RMD:`
+            - get the building segment swh_bat.  If it's "OTHER" increment, num_segments: `if (get_building_segment_SWH_bat(B_RMD, building_segment) == "OTHER"): num_segments += 1`
```

Comment:
I would need to go back and look at the get_building_segment_SWH_bat function but maybe you know, would this function have separate BATs if they have the same designation "OTHER", "UNDETERMINED" or would it be grouped together?